tests/settings.py

```diff
@@ -7,3 +7,5 @@
 ]
 
 SECRET_KEY = "spam-eggs"
+
+USE_TZ = True
```

Comment:
This fixes the Django 4.0+ warning:

```
django.utils.deprecation.RemovedInDjango50Warning: The default value of USE_TZ will change from False to True in Django 5.0. Set USE_TZ to False in your project settings if you want to keep the current default behavior.
```

ref https://docs.djangoproject.com/en/4.0/releases/4.0/#time-zone-support